src/components/Table/GridTable.stories.tsx

```diff
@@ -164,6 +165,8 @@ const rows: GridDataRow<NestedRow>[] = [
       },
       // Put this "grandchild" in the 2nd level to show heterogeneous levels
       { kind: "grandChild", id: "p1g1", name: "grandchild p1g1" },
+      // Put this "kind" into the 2nd level to show it doesn't have to be a card
+      { kind: "add", id: "add", pin: "last" },
```

Comment:
I went back/forth between putting the `pin` key directly into the `GridDataRow`, or putting in the `GridRowStyle` which is where most of our layout-ish settings (like indent, row links, etc.) goes...

I dunno, maybe it makes more sense to add in the `GridRowStyle`...